docs/Getting_Started.mdx

```diff
@@ -72,7 +72,7 @@ The [`config`](./packages/TypeScript_ESLint.mdx) helper sets three parts of ESLi
 
 - [Parser](https://eslint.org/docs/latest/use/configure/parser): set to [`@typescript-eslint/parser`](./packages/Parser.mdx) so ESLint knows how to parse the new pieces of TypeScript syntax.
 - [Plugins](https://eslint.org/docs/latest/use/configure/plugins): set to [`@typescript-eslint/eslint-plugin`](./packages/ESLint_Plugin.mdx) to load [rules listed in our _Rules_ page](./Rules).
```

Comment:
```suggestion
- [Plugins](https://eslint.org/docs/latest/use/configure/plugins): set to [`@typescript-eslint/eslint-plugin`](./packages/ESLint_Plugin.mdx) to load [rules listed in our _Rules_ page](/rules).
```

Let's fix one more broken link in this PR? 

https://github.com/typescript-eslint/typescript-eslint/issues/8480